webknossos-datastore/app/com/scalableminds/webknossos/datastore/models/datasource/DataLayerMapping.scala

```diff
@@ -0,0 +1,13 @@
+package com.scalableminds.webknossos.datastore.models.datasource
+
+import play.api.libs.json._
+
+/*
+Note: This case class is not (de)serialized to/from JSON using the build-in JSON library
+      but instead uses the dedicated MappingParser class for performance reasons.
+      Whenever this data class is changed, the parser needs to be modified accordingly.
+*/
+
+trait AbstractDataLayerMapping
```

Comment:
please add a comment about this empty trait being there to support putting these into a shared cache